### ISSUE TITLE
Assign one reviewer per sessionAssign one reviewer per session

### ISSUE THREAD

**Issue by kracekumar** [pythonindia/junction#380]

**Comment by ananyo2012** [comment]
@kracekumar Can you recall what is meant by a session here ?

**Comment by pradyunsg** [comment]
I think one reviewer for each talk/workshop session?

**Comment by ananyo2012** [comment]
You mean for each proposal section ? That's already there I believe. Each reviewer can be given review comment access for  specific proposal sections e.g. ML, Devops etc.

**Comment by pradyunsg** [comment]
No, as in every talk/workshop (AKA "session") proposal gets a single person assigned as a reviewer.

**Comment by ananyo2012** [comment]
Ah I see you mean each proposal by default gets a reviewer assigned

**Comment by kracekumar** [comment]
> No, as in every talk/workshop (AKA "session") proposal gets a single person assigned as a reviewer.

True. Not sure, if the organizer would want this as default behavior. The idea is to make sure all proposers get some feedback

**Comment by pradyunsg** [comment]
I see. I think that's more a workflow thing than something that Junction -- the tool -- should be handling.

Overall, I don't think there's anything actionable here; if there is please holler. :)
